settings.gradle.kts

```diff
@@ -27,29 +31,8 @@ dependencyResolutionManagement {
 
     // Catalogs
     versionCatalogs {
-        create("libs") {
-
-            // Kotlin & KotlinX
-            val kotlinxGroup = "org.jetbrains.kotlinx"
-            val coroutinesPrefix = "kotlinx-coroutines"
-
-            val coroutinesVersion = "1.6.4"
-
-            library("$coroutinesPrefix-core", kotlinxGroup, "$coroutinesPrefix-core").version(coroutinesVersion)
-            library("$coroutinesPrefix-reactor", kotlinxGroup, "$coroutinesPrefix-reactor").version(coroutinesVersion)
-            val kotlinVersion: String by settings
-            library("kotlin-reflect", "org.jetbrains.kotlin", "kotlin-reflect").version(kotlinVersion)
-        }
-
-        create("micronautLibs") {
-            version("micronaut", "4.0.5")
-            library("management", "io.micronaut", "micronaut-management").withoutVersion()
-            library("http", "io.micronaut", "micronaut-http").withoutVersion()
-            library("http-client", "io.micronaut", "micronaut-http-client").withoutVersion()
-        }
-
         create("testLibs") {
-            val kotestApiVersion = "5.5.5"
+            val kotestApiVersion = "5.6.2"
```

Comment:
Probably you did, but better to ask twice: did you check that this matches the version of the transitive kotest dep that comes from micronaut-kotest?

Reply:
jep